portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -1190,6 +1190,13 @@ definition {
 			locator1 = "FormFields#GRID_OPTIONS_OPTION_ROW_NAME");
 	}
 
+	macro viewOptionSingleSelection {
+		AssertElementPresent(
+			key_bodyText = "${bodyText}",
```

Comment:
@gleicelisbino, what is the `bodyText` of an option?

From what I remember, we have the option name, option value and its reference. Can you please clarify?

Reply:
@natocesarrego  When I created the path I use, for about this context, I remember I just kept it. But I agree, it doesn't make much sense.